ansible_collections/arista/avd/docs/getting-started/intro-to-ansible-and-avd.md

```diff
@@ -0,0 +1,438 @@
+
+Tasks in Ansible are executed based on structured data, written in a format called [YAML](https://yaml.org).
+
+YAML will be covered later in the section “How does Ansible Work”.
+
+Ansible can be used to automate many things. In this document, we’re only focusing on the use of Ansible to help us
+automate the provisioning of Arista network devices either directly from Ansible or via Arista CloudVision.
+
+## What is required to run Ansible
+Ansible can run on almost anything, but in production scenarios, we typically see Ansible deployed on a virtual Linux server,
+running on the customer’s preferred hypervisor. This Ansible server then communicates either directly with the
+Arista network devices or with Arista CloudVision Portal, which in turn communicates with the Arista network devices.
+Controlling what Ansible does is typically done using a SSH terminal session to the Ansible server from the Operator’s computer.
+Using Ansible to control an Arista network infrastructure is agentless,
+i.e. it requires nothing additional to be installed on the Arista network devices or on Arista CloudVision Portal and only uses the existing APIs to remotely configure the network.
+
+![Figure: Ansible and CVP](../_media/getting-started/Ansible-and-CVP-httpapi.png)
+
+## What is Ansible AVD (Ansible collection for Arista Validated Designs)
+While Ansible is the automation engine, Ansible AVD is a “collection” added to Ansible.
+It's an open source project hosted on GitHub and maintained by Arista, letting users contribute directly. 
+A “collection” is a distribution format for Ansible content that can include roles, modules, and plugins.
+These topics are most likely new to you, but do not worry, they will be described in later sections.
+For now, just remember that a “collection” is a way to easily share and distribute Ansible data.
+You can install and use collections through Ansible Galaxy, which is Ansible’s official hub for sharing Ansible content.
+
+![Figure: Arista Leaf Spine topology](../_media/getting-started/Ansible-AVD.png)
+
+In the illustration above, the Ansible AVD collection is shown as the box with the red background on the right hand side.
+
+When designing an EVPN Layer 3 Leaf Spine (L3LS) fabric you don't need to be an expert on Arista Design and Deployment Guides.
+All those rules and guidelines are integrated in the Ansible collection for Arista Validated Designs (AVD), or in short: Ansible AVD.
+
+Ansible AVD is an opinionated collection, meaning that field experiences and best practices from actual deployments at Arista customers are included.
+With Ansible AVD, users get up and running quickly without having to invent every part of the network configuration from scratch.
+This means only the most basic information must be provided by the user to get a new fabric up and running.
+ 
+Parameters which need to be defined by the user include:
```

Comment:
This sentence can confuse a new user to think these are all the required parameters. Consider rephrasing to mention that this list is not limited to all parameters required.